src/MomentoRedisClient.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Momento\Cache;
+
+use Exception;
+use Momento\Auth\EnvMomentoTokenProvider;
+use Momento\Cache\Errors\InvalidArgumentError;
+use Momento\Config\Configurations\Laptop;
+
+class MomentoRedisClient
```

Comment:
How do you see type safety working here with the existing Redis client? If there an interface we can implement or client we can derive from? Free to talk on slack as well.